pkg/filesystem/local_directory_test.go

```diff
@@ -13,12 +14,12 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-func openTmpDir(t *testing.T) filesystem.Directory {
+func openTmpDir(t *testing.T) (filesystem.Directory, string) {
```

Comment:
Out of curiosity: did you run into issues with these tests leaving files behind on disk? The number of files left behind is not significant and Bazel will clean it up for us, right?

Reply:
Bazel will clean up, but I was running the tests manually. (I was actually cross compiling on Linux and then executing the tests manually on Windows.)